docs/tutorials/rainbow-bridge-frontend.md

```diff
@@ -0,0 +1,153 @@
+---
+id: rainbow-bridge-frontend
+title: Using the Rainbow Bridge (MainNet)
+sidebar_label: Bridge Front-end walkthrough
+---
+
+## TutorPrerequisites 
+
+- $100 USD or equivalent currency _(some of which will be burnt in transfer fees)_
+- [NEAR Mainnet Wallet](/docs/develop/basics/create-account#creating-a-mainnet-account)
+- [MetaMask](https://metamask.io/download)
+
+**Note:** Rainbow Bridge does not currently support NEAR Wallets that are secured using a Ledger device.
+
+---
+
+## Step 1 - Buy Ether
+
+
+- Using the metamask extension purchase some ETH.
+
+
+<center> 
+
+![](https://i.imgur.com/zKC5VBe.png=400x) 
```

Comment:
add some alt data here in case of broken links. 
ex: ` ![metamask eth purchase](https://i.imgur.com/zKC5VBe.png=400x)`

Reply:
Added alt messages :)